calibration/NUTNRB/CGINS-NUTNRB-01065__20180319.csv

```diff
@@ -0,0 +1,8 @@
+NTR-1065,CC_lower_wavelength_limit_for_spectra_fit,217,constant
+NTR-1065,CC_upper_wavelength_limit_for_spectra_fit,240,constant
+NTR-1065,CC_wl,"[189.66, 190.46, 191.26, 192.06, 192.87, 193.67, 194.47, 195.28, 196.08, 196.89, 197.69, 198.49, 199.3, 200.1, 200.91, 201.71, 202.52, 203.33, 204.13, 204.94, 205.74, 206.55, 207.36, 208.16, 208.97, 209.78, 210.59, 211.39, 212.2, 213.01, 213.82, 214.62, 215.43, 216.24, 217.05, 217.86, 218.67, 219.48, 220.28, 221.09, 221.9, 222.71, 223.52, 224.33, 225.14, 225.95, 226.76, 227.57, 228.38, 229.2, 230.01, 230.82, 231.63, 232.44, 233.25, 234.06, 234.87, 235.68, 236.5, 237.31, 238.12, 238.93, 239.74, 240.56, 241.37, 242.18, 242.99, 243.81, 244.62, 245.43, 246.24, 247.06, 247.87, 248.68, 249.5, 250.31, 251.12, 251.94, 252.75, 253.56, 254.38, 255.19, 256.01, 256.82, 257.63, 258.45, 259.26, 260.08, 260.89, 261.7, 262.52, 263.33, 264.15, 264.96, 265.78, 266.59, 267.4, 268.22, 269.03, 269.85, 270.66, 271.48, 272.29, 273.11, 273.92, 274.74, 275.55, 276.37, 277.18, 278, 278.81, 279.63, 280.44, 281.26, 282.07, 282.88, 283.7, 284.51, 285.33, 286.14, 286.96, 287.77, 288.59, 289.4, 290.22, 291.03, 291.85, 292.66, 293.48, 294.29, 295.11, 295.92, 296.74, 297.55, 298.37, 299.18, 299.99, 300.81, 301.62, 302.44, 303.25, 304.07, 304.88, 305.69, 306.51, 307.32, 308.14, 308.95, 309.76, 310.58, 311.39, 312.21, 313.02, 313.83, 314.65, 315.46, 316.27, 317.09, 317.9, 318.71, 319.53, 320.34, 321.15, 321.96, 322.78, 323.59, 324.4, 325.21, 326.03, 326.84, 327.65, 328.46, 329.27, 330.09, 330.9, 331.71, 332.52, 333.33, 334.14, 334.95, 335.77, 336.58, 337.39, 338.2, 339.01, 339.82, 340.63, 341.44, 342.25, 343.06, 343.87, 344.68, 345.49, 346.3, 347.11, 347.91, 348.72, 349.53, 350.34, 351.15, 351.96, 352.76, 353.57, 354.38, 355.19, 355.99, 356.8, 357.61, 358.42, 359.22, 360.03, 360.84, 361.64, 362.45, 363.25, 364.06, 364.86, 365.67, 366.47, 367.28, 368.08, 368.89, 369.69, 370.5, 371.3, 372.1, 372.91, 373.71, 374.51, 375.32, 376.12, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.73, 382.53, 383.33, 384.13, 384.93, 385.73, 386.53, 387.33, 388.13, 388.93, 389.73, 390.53, 391.32, 392.12, 392.92, 393.72, 394.51, 395.31, 396.11]",
```

Comment:
Looks good.

Reply:
Looks good.  Add note to source file comment: "from Pre-Deployment".

Reply:
added note.